planet4-gutenberg-blocks.php

```diff
@@ -334,6 +334,67 @@ function ( $std_class_object ) {
 	return $twig;
 }
 
+add_filter( 'timber/twig', 'p4_optimize_img_url' );
+
+/**
+ * Add Twig filter(cf_img_url) which convert image url/s into cloudflare optimized image url/s.
+ *
+ * @param \Twig\Environment $twig The Twig environment.
+ * @return \Twig\Environment
+ */
+function p4_optimize_img_url( $twig ) {
+	// TO DO: For now keep the flag value(Cloudflare image optimization) hardcoded. later on we will make it configurable from settings.
+	$cf_img_optimization = true;
+
+	// Adding functions as filters.
+	$twig->addFilter(
+		new Twig_SimpleFilter(
+			'cf_img_url',
+			function ( $source, $srcset = '' ) use ( $cf_img_optimization ) {
+				if ( true !== $cf_img_optimization ) {
+					return $source;
+				}
+
+				// Cloudflare url format - https://zone/cdn-cgi/image/options/source-image .
+				// eg. <img src="/cdn-cgi/image/width=80,quality=75/uploads/avatar1.jpg">.
+				// More info. https://developers.cloudflare.com/images/about .
+				$zone       = '';
+				$prefix     = '/cdn-cgi/image/';
+				$options[0] = 'fit=contain';
```

Comment:
can you also try adding `format=auto`? According to the [documentation](https://developers.cloudflare.com/images/about) this will serve WebP images for the browsers that support it.